test/setup/setup.spec.js

```diff
@@ -0,0 +1,5 @@
+describe('setup', () => {
```

Comment:
Can you please explain why this file is necessary. is localStorage necessary for tests?